### DISCUSSION TITLE
Ability to prevent source string comments (at component level)

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#11196] [Ideas]
Hello,
I would like to have the ability to prevent source string comments, because the source strings are rather sensitive and should be elaborately discussed in context. Ideally, instead of allowing users to comment, I would like to have a button "Source string feedback" which opens a link.

Also, the comments are misused for other things than providing comments on the source string:
![Screenshot 2024-03-13 231051](https://github.com/WeblateOrg/weblate/assets/11229646/f18767c8-ba9d-42ad-b783-0cee3373f645)

I would like to be able to set this (enable [current, default]; link; disable) + the URL field at component level.

**Comment by nijel** [comment]
The built-in roles allow commenting when the user can translate, but this can be customized, see https://docs.weblate.org/en/latest/admin/access.html
